compiler/gentype/Paths.ml

```diff
@@ -29,17 +29,43 @@ let find_name_space cmt =
   cmt |> Filename.basename |> (Filename.chop_extension [@doesNotRaise])
   |> keep_after_dash
 
-let get_output_file_relative ~config cmt =
-  (cmt |> handle_namespace) ^ ModuleExtension.ts_input_file_suffix ~config
+let remove_project_root_from_absolute_path ~(config : Config.t) source_path =
```

Comment:
This is the bit that I haven't really reviewed properly yet.
Might not be a robust way to get a relative path. Especially Windows vs  linux etc.
Can't remember if the code already does this somewhere else.
Or, poking around in the Filename module, there's more one can use.

Reply:
I refactored the function to use `Filename.concat` for appending the directory separator when missing. Additionally, it now verifies if the prefix exists and returns the original path if not. I also renamed the function for better clarity.